solr/core/src/test/org/apache/solr/handler/TestContainerPlugin.java

```diff
@@ -509,6 +541,24 @@ public void m2(SolrQueryRequest req, SolrQueryResponse rsp) {
     }
   }
 
+  public static class ConfigurablePluginWithValidation
+      implements ConfigurablePlugin<ValidatableConfig> {
+    @Override
+    public void configure(ValidatableConfig cfg) {
+      cfg.validate();
+    }
+  }
+
+  public static class ValidatableConfig implements ReflectMapWriter {
+    @JsonProperty public boolean willPassValidation;
+
+    public void validate() {
```

Comment:
Can this be merged into `ConfigurablePluginWithValidation#configure`? Again, just to make the code easier to follow by avoiding one jump.

Reply:
Sure, it can definitely be implemented this way :

```java
public static class ConfigurablePluginWithValidation implements ConfigurablePlugin<ValidatableConfig> {
  @Override
  public void configure(ValidatableConfig cfg) {
    if (!cfg.willPassValidation) {
      throw new SolrException(SolrException.ErrorCode.BAD_REQUEST, "invalid config");
    }
  }
}

public static class ValidatableConfig implements ReflectMapWriter {
  @JsonProperty public boolean willPassValidation;
}

```

Is it what you have in mind ?

Reply:
Yes. Very minor comment, I just thought it would slightly improve the readability of the test. ;)

Reply:
Thanks, I updated the code accordingly in https://github.com/apache/solr/pull/2202/commits/729b14b191f79fc99759f94cdd3a45a09e7ffa2c